schema/vulntology-json-schema-1.0-draft.json

```diff
@@ -370,7 +378,8 @@
             "type": "object",
             "required": [
                 "hasIdentity",
-                "hasScenario"
+                "hasScenario",
+                "hasOriginatingProduct"
```

Comment:
This reverses a prior PR (usnistgov/vulntology#73) where we reduced required properties.